### PR TITLE
Dropup icon added

### PR DESCRIPTION
**Pull request by satyamkale27**
## Description
 dropup icon  added 

#### Issue addressed
<#594>

Addresses #*PR# HERE*

### Before/after screenshots
Before
before

 After
after

## Changelog

- [#PR no]
  - **Description:** added a new svg icon dropup-icon
  
[#PR no]: PR link

## Steps to test

1. Step 1 run the project yarn dev
2. Step 2   go to http://localhost:4000/icons#token-dropup
3. ...

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [x] If there are any front-end changes, before/after screenshots are included
- [] Critical and brittle code paths are covered by unit tests
- [] The change is described in the changelog section above

## Reviewer guidance

- [x] Is the code clean and well-commented?
- [ ] Are there tests for this change?
- [ ] Are all UI components LTR and RTL compliant (if applicable)?
- [ ] _Add other things to check for here_

## After review

- [ ] The changelog item has been pasted to the `CHANGELOG.md`

### CODE REVIEW

**Comment by satyamkale27** [comment]
@MisRob take a look

File: lib/KIcon/iconDefinitions.js [hunk 1]
```diff
@@ -29,6 +29,9 @@ const KolibriIcons = {
   dropdown: {
     icon: require('./precompiled-icons/material-icons/arrow_drop_down/baseline.vue').default,
   },
+  dropup: {
+    icon: require('./precompiled-icons/material-icons/arrow_drop_up/baseline.vue').default,
```
Comment (MisRob) [inline lib/KIcon/iconDefinitions.js]: The path here needs to be `precompiled-icons/le/dropup.vue` as that's the path where the `.vue` file was generated for the svg you added

Comment (satyamkale27) [reply lib/KIcon/iconDefinitions.js]: done, made the changes.

**Review by MisRob — APPROVED** [review]
Thank you @satyamkale27, it looks good overall.

I pushed some changes related to rebasing to the `release-v3` branch. It's related to our release organization and nothing to be worried about.

I also updated CHANGELOG.md. If you'd like to keep contributing to the KDS, you could look at the CHANGELOG.MD changes I pushed and use the same approach in your next PRs.

**Pull request merged**